icekit/project/settings/_base.py

```diff
@@ -580,6 +580,7 @@
     'icekit.response_pages',
     'notifications',
 
+    'icekit.authors',
```

Comment:
It might be worth putting the `Author` model in here, and the corresponding list page in `icekit.authors.page_types.list_page` (with an `app_label` of `icekitauthors_pagetypes_listpage` or something). Currently, installing `icekit.authors` will always register the listing page.

Or better yet, since we are providing an authors list page and a separate (non-page-type) but related model to store the author details, we should move the whole authors app to `icekit.page_types.author`. Articles is a better candidate for a top level icekit package because it's only providing abstract functionality, not a concrete implementation. The authors app is a concrete implementation of the article pattern and provides/relies on a page type as a critical component.